modules/extensions/src/views/lite/components/debugger/index.tsx

```diff
@@ -259,9 +259,26 @@ export class Debugger extends React.Component<Props, State> {
     return <SplashScreen />
   }
 
+  renderProcessingTab() {
+    const processing = _.get(this.state, 'event.processing') as _.Dictionary<sdk.IO.ProcessingEntry> | null
+    if (!processing) {
+      return
+    }
+
+    const hasError = Object.values(processing).some(item => item.errors?.length > 0)
+
+    return (
+      <Tab
+        id="processing"
+        className={hasError && style.tabError}
```

Comment:
Prefer using classnames for conditional style: `cx({ [style.tabError]: hasError})`